eo2js/test/commands/transpile.test.js

```diff
@@ -65,9 +73,28 @@ describe('transpile', function() {
     });
     ['simple-test', 'alone-test'].forEach((name) => {
       it(`should generate test JS file for ${name}`, function() {
-        assertFilesExist(transpile(name), target, [`project/com/eo2js/${name}.test.js`])
+        prepare(name)
```

Comment:
@HaidarJbeily7 it became a little bit messy because `prepare` is called twice in `beforeEach` and in the test. I think the design of the helper functions here  should be changed somehow

Reply:
@maxonfjvipon I enhanced the helper functions a little bit. Please check again!